### DISCUSSION TITLE
Rework some things

### DISCUSSION THREAD

**Discussion by Phantaplasm** [FakeFishGames/Barotrauma#12187] [Ideas and suggestions]
The HMG is the most broken gun rn and it overshadows a lot of other guns like scrap cannon and ark emitter
Tell me why would anyone pick any of this guns if you can farm weapon skill, take the hmg and go destroy an entire tiger thresher swarm 
The ammo for hmg is really cheap also, so you can craft like 12 of them and don't even care, you can swim to the outpost without a sub 
I was playing in a server and one guy, just crafted an smg and obliterated all sub just because he had some magnesium with him 
Nerf HMG and buff other handheld weapons

**Comment by BiffW** [comment]
Yeah, I was thinking about this too. So I made this [mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2807919770) you might want to try out.

**Comment by Arctodites** [comment]
Some of the more powerful handheld weapons should get debuffs when firing them underwater as opposed to indoors: things like reduced accuracy, shorter effective range, chance to misfire (jams / dud rounds / fumbled reloads), slower rate of fire.  It would make for some interesting tradeoffs when venturing out on away missions; you'd have to think about packing a less powerful sidearm for backup -- a revolver or a harpoon or something, which would increase the tension of underwater encounters and keep early game weapons from becoming obsolete later on.

**Reply by Hazmat616** [reply]
Yeah the harpoon guns should always be THE weapon of choice to use in water.

**Comment by bookslayer10** [comment]
Well, you can't use it if you haven't farmed weapon skill, so it likely won't be usable for anyone other than the high level security guard who made it. Sure, everyone can grind weapons skill to use it eventually, but it will take a while, especially for lobbies with random players.

For players who only play a lobby for a few hours then leave, the only way to use the HMG is if someone else makes an Exosuit for them to wear as well, but the downside of that is the fuel rod consumption.

HMGs also slow your character down when carrying them in your hands, and all heavy weapons force you to not bring a toolbelt due to needing to store them on your back. Just bringing a riot shotgun with explosive slugs on a dive can be better than an HMG if you don't expect to need to fight many enemies.

Other heavy weapons like the scrap cannon, arc emitter, fusion gun, autoshotgun, and assault rifle all aren't as good as the HMG for a max level security officer, but they all are good if you are a new character without weapon skill or somebody who can craft HMGs.
